components/ota/index.rst

```diff
@@ -154,48 +136,32 @@ will contain one of values for the ``OTAState`` enum. These values are:
 .. code-block:: yaml
 
     ota:
-      on_state_change:
-        then:
-          - if:
-              condition:
-                lambda: return state == ota::OTA_STARTED;
-              then:
-                - logger.log: "OTA start"
-
-Updating the Password
----------------------
-
-Changing an Existing Password
-*****************************
-
-Since the configured password is used for both compiling and uploading, the regular ``esphome run <file>`` command
-won't work. This issue can be worked around by executing the operations separately with an ``on_boot`` trigger:
-
-.. code-block:: yaml
+      - platform: ...
+        on_state_change:
+          then:
+            - if:
+                condition:
+                  lambda: return state == ota::OTA_STARTED;
+                then:
+                  - logger.log: "OTA start"
+
+Safe Mode
+---------
 
-    esphome:
-      on_boot:
-        - lambda: |-
-            id(my_ota).set_auth_password("New password");
-
-    ota:
-      - platform: esphome
-        id: my_ota
-        password: "Old password"
-
-After this trick has been used to change the password, the ``on_boot`` trigger may be removed and the old password
-replaced with the new password in the ``ota:`` section.
-
-Adding a Password
-*****************
-
-If OTA is already enabled without a password, simply add a ``password:`` line to the existing ``ota:`` config block.
+In addition to OTA updates, ESPHome also supports a "safe mode" to help with recovery if/when updates don't work as
+expected. This is automatically enabled by the ``ota`` component, but it may be disabled if desired. See
+:doc:`/components/safe_mode` for details.
 
 See Also
 --------
 
 - :apiref:`ota/ota_component.h`
-- :doc:`/components/button/safe_mode`
-- :doc:`/components/switch/safe_mode`
-- :doc:`/components/ota_http_request`
+
+.. toctree::
+    :maxdepth: 1
+    :glob:
+
+    *
```

Comment:
```suggestion

Platforms
---------

.. toctree::
    :maxdepth: 1
    :glob:

    *

See Also
--------

- :apiref:`ota/ota_component.h`
```

Reply:
Couldn't apply here, but done in [`aa78f73` (#3976)](https://github.com/esphome/esphome-docs/pull/3976/commits/aa78f7303b361728eced25348066e4f00a711c61)